source/client/client_worker_impl.h

```diff
@@ -34,23 +37,26 @@ class ClientWorkerImpl : public WorkerImpl, virtual public ClientWorker {
   const std::map<std::string, uint64_t>& thread_local_counter_values() override {
     return thread_local_counter_values_;
   }
-  const Sequencer& sequencer() const override { return *sequencer_; }
+
+  const std::vector<PhasePtr>& phases() const override { return phases_; }
```

Comment:
Can we document the new public method?

Reply:
This one is now renamed. Doc comments related to this reside in the interface class, does that suffice?